build.gradle

```diff
@@ -800,9 +800,9 @@ artifactoryPublish {
   dependsOn distZip
 }
 
-def dockerBuildVersion = project.hasProperty('release.releaseVersion') ? project.property('release.releaseVersion') : "${rootProject.version}"
-def dockerOrgName = project.hasProperty('dockerOrgName') ? project.getProperty("dockerOrgName") : "hyperledger"
-def dockerArtifactName = project.hasProperty("dockerArtifactName") ? project.getProperty("dockerArtifactName") : "besu"
+def dockerBuildVersion = project.hasProperty('release.releaseVersion') && project.property('release.releaseVersion') ? project.property('release.releaseVersion') : "${rootProject.version}"
```

Comment:
What does this line achieve? `project.hasProperty('release.releaseVersion') && project.property('release.releaseVersion')`? @macfarla

Reply:
project.hasProperty is does it exist
project.property will give true if it's not null

kind of like getting a value from a java Optional

Reply:
basically this set of changes was to make sure docker org name and artifact name never get set to empty